swimrankings.py

```
@@ -1,136 +1,215 @@
 
-def get_swimmer(full_name: str):
-    """
-    Scrapes swimrankings to find data about a swimmer based on a comma seperated name that is provided
+                gender = self.get_swimmer_gender(athlete_id)
 
-    Side effects:
-    - Scrapes swimrankings.net. Many calls to this function (from experience 31 requests in less than a second) in a short period of time can cause swimrankings.net to sh*t its pants and crash.
-        The reason for this is because its server os is Windows. lmaoo
+                return [athlete_id, birth_year, first_name, last_name, gender]
+            
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)